addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -15,25 +15,25 @@
                         <div t-attf-class="col-12 col-lg-9 d-flex flex-column #{'offset-lg-3' if slide.channel_id.channel_type == 'training' else ''}">
                             <h2 class="font-weight-medium w-100">
                                 <a t-att-href="'/slides/%s' % (slug(slide.channel_id))" class="text-white text-decoration-none" t-field="slide.channel_id.name"/>
-                                <t t-if="slide.channel_id.completed">
-                                    <small><span class="badge badge-pill badge-success pull-right my-1 py-1 px-2 font-weight-normal"><i class="fa fa-check"/> Completed</span></small>
-                                </t>
+                                <small t-attf-class="o_wslides_channel_completion_completed #{'d-none' if not slide.channel_id.completed else ''}">
+                                    <span class="badge badge-pill badge-success pull-right my-1 py-1 px-2 font-weight-normal"><i class="fa fa-check"/> Completed</span>
+                                </small>
                             </h2>
 
                             <div t-if="slide.channel_id.channel_type == 'documentation'" class="mb-3 small">
                                 <span class="font-weight-normal">Last update:</span>
                                 <t t-esc="slide.date_published" t-options="{'widget': 'date'}"/>
                             </div>
 
-                            <div t-else="" t-if="not slide.channel_id.completed" class="d-flex align-items-center pb-3">
+                            <div t-else="" t-attf-class="o_wslides_channel_completion_progressbar #{'d-none' if slide.channel_id.completed else 'd-flex'} align-items-center pb-3">
                                 <div class="progress w-50 bg-black-25" style="height: 10px;">
                                     <div class="progress-bar rounded-left" role="progressbar"
                                         t-att-aria-valuenow="slide.channel_id.completion" aria-valuemin="0" aria-valuemax="100"
                                         t-attf-style="width: #{slide.channel_id.completion}%;">
                                     </div>
                                 </div>
-                                <i t-att-class="'fa fa-trophy m-0 ml-2 p-0 %s' % ('text-warning' if slide.channel_id.completed else 'text-black-50')"></i>
-                                <small class="ml-2 text-white-50"><t t-esc="slide.channel_id.completion"/> %</small>
+                                <i class="fa fa-trophy m-0 ml-2 p-0 text-black-50"></i>
+                                <small class="o_wslides_progress_percentage ml-2 text-white-50"><t t-esc="slide.channel_id.completion"/> %</small>
```

Comment:
The percentage should be outside `<small>` (it would disappear after an update)

Reply:
@jeh-odoo good catch :)